src/search/FindUtils.js

```diff
@@ -352,11 +387,209 @@ define(function (require, exports, module) {
         return {valid: true, queryExpr: queryExpr};
     }
 
+     /**
+     * Prioritizes the open file and then the working set files to the starting of the list of files
+     * @param {Array.<*>} files An array of file paths or file objects to sort
+     * @param {?string} firstFile If specified, the path to the file that should be sorted to the top.
+     * @return {Array.<*>}
+     */
+    function prioritizeOpenFile(files, firstFile) {
+        var workingSetFiles = MainViewManager.getWorkingSet(MainViewManager.ALL_PANES),
+            workingSetFileFound = {},
+            fileSetWithoutWorkingSet = [],
+            startingWorkingFileSet = [],
+            propertyName = "",
+            i = 0;
+        firstFile = firstFile || "";
```

Comment:
The line is not part of the var statement, so the indentation is correct in this case.